internal/manifests/manifests_test.go

```diff
@@ -191,6 +191,95 @@ var _ = Describe("ClusterManifestTests", func() {
 			Expect(err.StatusCode()).To(Equal(int32(http.StatusBadRequest)))
 			Expect(err.Error()).To(ContainSubstring("failed to base64-decode cluster manifest content"))
 		})
+
+		Context("File extension and format", func() {
+			It("accepts manifest in json format and .json extension", func() {
+				clusterID := registerCluster().ID
+				jsonContent := "eyAiaGVsbG8iOiAid29ya2QiIH0K"
```

Comment:
I think it would be nicer to have the real content and you just base64 encode it.